ci/delete-temp-images.sh

```diff
@@ -0,0 +1,46 @@
+# Authenticate and retrieve DockerHub token
```

Comment:
Same here, this would benefit from some strictness so it'll fail faster, and nearer to the source of any issues.

```bash
#!/bin/bash

set -e -E -u -o pipefail
```